### ISSUE TITLE
Please create the nextcloud app for ubuntu touch OS

### ISSUE THREAD

**Issue by ghost** [nextcloud/desktop#4614]
Please create the nextcloud app for ubuntu touch OS. https://ubuntu-touch.io, https://ubports.com, https://open-store.io and https://clickable-ut.dev are links for more information about ubuntu touch OS.

**Comment by BEEDELLROKEJULIANLOCKHART** [comment]
By create, do you mean develop, from nothing, new software specifically for Ubuntu Touch, for to convert it so that it operates or presents its interface correctly? (Don't respond with yes or no: specify.)

**Comment by apy7hst** [comment]
A new, official nextcloud application for Ubuntu Touch is what I meant.

**Comment by BEEDELLROKEJULIANLOCKHART** [comment]
@apy7hst, why? Is the current software inadequate somehow? Creation of entirely new software is difficult to commit to without obvious rationale, and demonstration of why such investment is more advantageous than disadvantageous, due to the additional burden of initial development and maintenance.

**Comment by apy7hst** [comment]
Ok, then convert the software so that it works on ubuntu touch but officially. The UI of nextcloud for Android / iOS and the nextcloud linux application can be a perfect combination for ubuntu touch OS

**Comment by BEEDELLROKEJULIANLOCKHART** [comment]
I agree. Replacement of the current interface, which appears to utilize Qt, with Kirigami would provide the desired adaptability.

**Comment by apy7hst** [comment]
Therefore, are you ready to create the nextcloud app for ubuntu touch ?

**Comment by claucambra** [comment]
I would say this is sort of a duplicate of #2934 -- I will close in favour of that issue
